services/web/server/src/simcore_service_webserver/products/_db.py

```diff
@@ -105,6 +106,13 @@ async def get_product_latest_credit_price_or_none(
             )
             return parse_obj_as(NonNegativeDecimal | None, usd_per_credit)
 
+    async def get_product_stripe_info(self, product_name: str) -> ProductStripeInfoGet:
+        async with self.engine.acquire() as conn:
+            row = await get_product_latest_stripe_info(conn, product_name=product_name)
+            return ProductStripeInfoGet(
+                stripe_price_id=row[0], stripe_tax_rate_id=row[1]
```

Comment:
Prefer names pover indices. It is far less error-prone, i.e. `stripe_price_id=row.strip_price_id` etc

Reply:
hmm, I remember I tried it but it didn't work here for some reason, let me double check that